src/Numerics/Mesh/Topologies.jl

```diff
@@ -735,7 +783,190 @@ function StackedBrickTopology(
     nb = length(bndytoelem)
 
     T = eltype(basetopo.elemtocoord)
-
+    #----setting up DSS----------
+    if basetopo.elemtouvert isa Array
+        DA = CUDA.has_cuda_gpu() ? CuArray : Array
+        #---setup vertex DSS
+        nvertby2 = Int(nvert / 2)
```

Comment:
```suggestion
        nvertby2 = div(nvert, 2)
```
Should we be using `div` here?